files/leaflet.esri.renderers/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Rachel Nehmer"
+github = "https://github.com/Esri/esri-leaflet-related.git"
```

Comment:
fixed and rebased.  thanks!

someday i'll submit a PR here that doesn't have an immediately obvious problem :smile: